### PR TITLE
added picker + example

### PR DESCRIPTION
**Pull request by danielhorv**
Btw. CocoaPods is not working yet.

### CODE REVIEW

File: DHCountryPicker/CountryPickerViewController.swift [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+        
+        tableView.dataSource = self
+        tableView.delegate = self
+        searchController.searchResultsUpdater = self
+        
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShow), name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillHide), name: UIResponder.keyboardWillHideNotification, object: nil)
```
Comment (julianpomper) [inline DHCountryPicker/CountryPickerViewController.swift]: could be at least row 65-67 in the init?

File: DHCountryPicker/CountryPickerViewController.swift [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    }
+}
+
+extension CountryPickerViewController: UISearchResultsUpdating {
+    public func updateSearchResults(for searchController: UISearchController) {
+        let userInput = searchController.searchBar.text ?? ""
+        
+        if !userInput.isEmpty {
+            let filteredCountries = countries
+                .filter { $0.localizedName.lowercased().starts(with: userInput.lowercased()) }
```
Comment (julianpomper) [inline DHCountryPicker/CountryPickerViewController.swift]: maybe trim the input

File: DHCountryPicker/CountryProvider.swift [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            let data = try Data(contentsOf: countryCodesJson)
+            let jsonDecoder = JSONDecoder()
+            availableCountries = try jsonDecoder.decode([Country].self, from: data)
+                .sorted { $0.localizedName < $1.localizedName }
+        } catch {
+            print(error)
+        }
+    }
+    
+    public func current() -> Country? {
```
Comment (julianpomper) [inline DHCountryPicker/CountryProvider.swift]: computed variable?

File: DHCountryPicker/CountryPickerViewController.swift (hunk 1 above)
Comment (danielhorv) [reply DHCountryPicker/CountryPickerViewController.swift]: yop, done

File: DHCountryPicker/CountryPickerViewController.swift (hunk 2 above)
Comment (danielhorv) [reply DHCountryPicker/CountryPickerViewController.swift]: thx, done

File: DHCountryPicker/CountryProvider.swift (hunk 3 above)
Comment (danielhorv) [reply DHCountryPicker/CountryProvider.swift]: done

**Review by julianpomper — APPROVED** [review]

**Pull request merged**